src/app/merge-diff/merge-diff-adapter/merge-diff-adapter.service.ts

```diff
@@ -27,8 +29,15 @@ import { Injectable } from '@angular/core';
 })
 export class MergeDiffAdapterService {
 
-  constructor() { }
+  source: any;
 
+  constructor(private resources : MdmResourcesService) { }
+
+  loadCatalogueItem(id: string, domainType: ModelDomainRequestType)
+  {
+   return this.resources[domainType.toString()].get(domainType,id);
```

Comment:
Perhaps `ModalDomainRequestType` has the correct names for `MdmResourcesService` properties, but the safest way I've found is to use the `ElementTypesService.getBaseTypeForDomainType()` function to return a type object that contains the correct resource property name.